src/providers/fault-summary/cat-adi-part2/fault-summary.cat-adi-part2.ts

```diff
@@ -9,7 +9,7 @@ import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
 // import {
 //   manoeuvreCompetencyLabels as manoeuvreCompetencyLabelsCatAdiPart2,
 //   manoeuvreTypeLabels as manoeuvreTypeLabelsCatAdiPart2,
-// } from '../../../shared/constants/competencies/catb-manoeuvres';
+// } from '../../../shared/constants/competencies/catadi2-manoeuvres';
```

Comment:
Can we uncomment this, or get rid of it all together?

Reply:
Fixed in the next PR.